jupyterkernel/cadabra2_jupyter/kernel.py

```diff
@@ -0,0 +1,97 @@
+import ipykernel.kernelbase
+import sys
+import traceback
+
+# only additional cadabra2 C++ function that required wrapping
+import cadabra_translator
```

Comment:
Function that needs wrapping in pybind11. Name can be changed to fit in the current `cadabra2` namespace.